src/routes/write-exercise/components/Feedback.tsx

```diff
@@ -56,18 +56,14 @@ const Feedback = ({ documentWithResult, submission, needsToBeRepeated }: Feedbac
     message = `${getLabels().exercises.write.feedback.almostCorrect1} „${submission}“ ${
       getLabels().exercises.write.feedback.almostCorrect2
     }`
-  } else if (!submission) {
-    Icon = CloseCircleIconBold
-    background = BannerRed
-    message = wrongWithCorrectSolution
   } else {
     Icon = CloseCircleIconBold
     background = BannerRed
-    message = needsToBeRepeated
-      ? `${getLabels().exercises.write.feedback.wrong}\n${wrongWithCorrectSolution}`
-      : wrongWithCorrectSolution
+    message =
+      needsToBeRepeated && !submission
```

Comment:
Doesn't this have to be an or?
```suggestion
      needsToBeRepeated || !submission
```

Reply:
only the first one function as intended

Reply:
sorry, can you elaborate a little what you mean here? I am relatively sure it has to be a `||` here. We want to show the solution either if the user clicked on show solution (`!submission`) or the user entered it too many times (`!needsToBeRepeated`).

Reply:
yes, you are right. Logically it fits more when we say needsToBeRepeated || !submission. It works.

Reply:
It appears that the problem is in InteractionSection, needsToBeSubmitted was first declared there. Input and button becomes disabled once the word is entered and there is no chance to enter and to check a given word

Reply:
Hi Steffen! needsToBeRepeated || !submission returns only true, but when I  use needsToBeRepeated && !submission, it both expressions are met successfully

Reply:
Hey, sorry but I don't understand what you mean, could you elaborate a little bit more please?

Reply:
**needsToBeRepeated || !submission** are validating only the first expression in ternary operator, but we need to say that if the user's input is incorrect, show an error message with correct answer. Otherwise show **correct answer**. 
**needsToBeRepeated && !submission** are validating both expressions in ternary operator. It is validating correctly.

Hope I was able to deliver my thoughts precisely:)

Reply:
Ahh now I get it, you changed the if with the else. Before the if was `${getLabels().exercises.write.feedback.wrong}\n${wrongWithCorrectSolution}` and the else was `wrongWithCorrectSolution`, now it's the other way round. Yes, then of course it is correct as it is.

Reply:
I think we don't have to check for `needsToBeRepeated` at all. We should show that the answer is wrong in any case when an anser was entered. So it should be:
``` ts
    message = submission
      ? `${getLabels().exercises.write.feedback.wrong}\n${wrongWithCorrectSolution}`
      : wrongWithCorrectSolution
```
Please mind: I changed the order back because positive conditions are easier to read then negative ones. Then you can also remove `needsToBeRepeated` from the props.